src/Test.sol

```diff
@@ -14,18 +14,12 @@ contract SuaveEnabled is Test {
     ConfidentialInputsWrapperI constant confInputsWrapper = ConfidentialInputsWrapperI(Suave.CONFIDENTIAL_INPUTS);
 
     function setUp() public {
-        string[] memory inputs = new string[](3);
+        string[] memory inputs = new string[](2);
         inputs[0] = "suave-geth";
         inputs[1] = "forge";
-        inputs[2] = "status";
 
-        try vm.ffi(inputs) returns (bytes memory response) {
-            // the status call of the `suave-geth forge` command fails with the 'not-ok' prefix
-            // which is '6e6f742d6f6b' in hex
-            if (isPrefix(hex"6e6f742d6f6b", response)) {
-                revert("Local Suave node not detected running");
-            }
-        } catch (bytes memory reason) {
+        try vm.ffi(inputs) returns (bytes memory response) {}
```

Comment:
The `suave-geth forge status` had two goals:
- Figure out if the `suave-geth` binary was available on PATH.
- Check if there was a running instance of `suave-geth` to resolve the precompiles (`status` subcommand).

Now, with the headless execution, we can skip step 2.